runtime/vm/createramclass.cpp

```diff
@@ -1250,7 +1253,7 @@ processVTableMethod(J9VMThread *vmThread, J9ClassLoader *classLoader, UDATA *vTa
 						}
 						/* fill in vtable, override parent slot */
 						J9ClassLoader *superclassVTableMethodLoader = superclassVTableMethodClass->classLoader;
-						if (superclassVTableMethodLoader != classLoader) {
+						if ((superclassVTableMethodLoader != classLoader) && verifierLoaded) {
```

Comment:
Please move the check out to include line 1255 as well.   I'd rather deal with an extra level indenting if it gives simpler to reason about code

Reply:
Already adjusted as suggested.